models/Card.js

```diff
@@ -0,0 +1,52 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+
+// create our Post model
+class Card extends Model {}
+
+Card.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    user_id: {
```

Comment:
Remove user id. No relation between the two. A user can own a deck, but not a card.